instrumentation/jetty-httpclient/jetty-httpclient-9.0/javaagent/jetty-httpclient-9.0-javaagent.gradle

```diff
@@ -0,0 +1,27 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.eclipse.jetty"
+    module = 'jetty-client'
+    versions = "[9.1,10)"
+    assertInverse = true
+  }
+}
+
+def jettyVers = [:]
+
+//Jetty client 9.1 is the best starting point for the v9.x series, v9.0.x is incompatible
+jettyVers.base9 = '9.1.0.v20131115'
+//jettyVers.base9 = '9.0.0.v20130308'
+jettyVers.useInTests9 = '9.3.28.v20191105'
```

Comment:
Thanks @anuraaga, this is definitely a question I had.  But the reason for the `useInTest9` version here is related to the jetty-server as I use it further down.  The testing infrastructure uses jetty-server as the backend for http client tests.   I needed to lock down the jetty-server at a version that doesn't conflict.  It seems the testing infra pulls in jetty-server-8.0 which the doesn't work when jetty-9 classes are being used in the main line test.

Reply:
Oh I completely missed that it's jetty-server, not client. I think we'll stil need that `latestDepTestLibrary` to restrict the client to 9.x (though have you confirmed it doesn't work on 10+?)

Reply:
Asking the same question as below:  still unsure how to use -PtestLatestDeps=true.  to cover testing in jetty-httpclient libraries 10+

Reply:
When running with `-PtestLatestDeps=true` version used for `library` is replaced with latest available version. If latest available doesn't work for some reason you can limit it by adding `latestDepTestLibrary "org.eclipse.jetty:jetty-server:10.+"` so instead of 11 it would use latest from 10 series. If you want to run tests on more versions then you can use `testSets` for example see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/hibernate/hibernate-4.3/javaagent/hibernate-4.3-javaagent.gradle though using testSets tends to be a hassle.

Reply:
@anuraaga A non-javaagent entrypoint is here now.   See JettyClientTracing.    
```
HttpClient client = JettyClientTracing.create()
//or
client = JettyClientTracing.from(client)
```

Reply:
Having trouble getting -PtestLatestDeps=true to work because the underlying testing framwork brings in Jetty-server but -PtestLatestDeps=true brings in jetty-server classes that are incompatible with underlying testing framework.